src/main/java/io/stargate/sgv2/jsonapi/service/processor/MeteredCommandProcessor.java

```diff
@@ -110,7 +115,49 @@ private Tags getCustomTags(Command command, CommandResult result) {
           (String) result.errors().get(0).fields().getOrDefault("errorCode", UNKNOWN_VALUE);
       errorCodeTag = Tag.of(jsonApiMetricsConfig.errorCode(), errorCode);
     }
-    Tags tags = Tags.of(commandTag, tenantTag, errorTag, errorClassTag, errorCodeTag);
+
+    Tag vectorEnabled =
+        commandContext.isVectorEnabled()
+            ? Tag.of(jsonApiMetricsConfig.vectorEnabled(), "true")
+            : Tag.of(jsonApiMetricsConfig.vectorEnabled(), "false");
+    JsonApiMetricsConfig.SortType sortType = getVectorTypeTag(command);
+    Tag sortTypeTag = Tag.of(jsonApiMetricsConfig.sortType(), sortType.name());
```

Comment:
Vector type is extract as enum but really used as `String`. So not sure intermediate enum is needed.

Reply:
I have made it as enum so the list of values used for this tag is controlled. Any other options you are suggesting?

Reply:
Ah. Ok, that makes sense then.